numasched/README.md

```diff
@@ -0,0 +1,61 @@
+# numasched
+
+The scheduling of system processes between NUMA nodes is tracked.
```

Comment:
Out of curiosity, is there a good use case for using this tool that can be in the readme?

Reply:
Just like bpftrace numaid commit https://github.com/bpftrace/bpftrace/commit/121f73df1cb873eecc940fc26c3dd0d8e0d272a2 said, numaid could help to optimize performance, This tool can help track the processes that migrate between NUMAs, thereby informing engineers to optimize